docs/index.md

```diff
@@ -0,0 +1,19 @@
+## Helm3 Charts for Sonatype Products
+
+These charts are designed to work out of the box with minikube using both Ingess 
```

Comment:
misspeel `Ingess`

Reply:
that's what i get for moving all the files. :) I own all the mistakes now!

Reply:
fixed